tools/topology/topology2/include/components/widget-common.conf

```diff
@@ -149,8 +149,22 @@ DefineAttribute."stack_bytes_requirement" {
 	token_ref	"comp.word"
 }
 
-## Heap size requirement in bytes for this component. Zero indicates default heap size.
-DefineAttribute."heap_bytes_requirement" {
+## Interim Heap size requirement in bytes for this component.
```

Comment:
used for resources that may change in size or be freed during the lifetime of the component.

Reply:
you could state kcontrols, fast get/put(), as examples.